client/plugins/browserDetection.ts

```diff
@@ -277,7 +297,15 @@ class BrowserDetector {
    * @memberof BrowserDetector
    */
   isMobile(): boolean {
-    if (/(?:iPhone|iPad|iPod|Android)/.test(navigator.userAgent)) {
+    let userAgent = '';
+
+    if (process.client && navigator.userAgent) {
+      userAgent = navigator.userAgent;
+    } else if (this.context.req.headers['user-agent']) {
+      userAgent = this.context.req.headers['user-agent'];
+    }
+
+    if (/(?:iPhone|iPad|iPod|Android)/.test(userAgent)) {
```

Comment:
We can remove this altogether and use Vuetify breakpoint

Reply:
They serve different purposes.

This is for platform detection, in order to know what we're running on, which we need in some cases (Like to fix some codec inconsistencies).

The Vuetify breakpoint only cares about viewport size, which isn't what we want here.